reviews/review-01+2020-11-02.md

```diff
@@ -0,0 +1,305 @@
+
+For example, chapter five could be:
+
+a)  Git at the Command Line (current title)
+
+b)  Tracking Changes
+
+c)  Tracking Changes with Git at the Command Line
+
+For chapter eight:
+
+a)  Make
+
+b)  Automating Analyses (current title)
+
+c)  Automating Analyses with Make
+
+It may not be possible to fully standardize the titles (I can't think of
+a good short punchline for the Unix shell, for example), but I think a
+review of the chapter titles to make them align more with the b) and c)
+options would be helpful in clarifying the purpose and big picture topic
+of each chapter.
+
+## Examples and exercises
+
+Are the chapter examples and exercises appropriate to the subject
+matter? If not, how can they be modified?
+
+Yes. One of the book's strengths is the presence of an intriguing and
+accessible sample problem that helps motivate the chapter topics and
+provides a concrete motivation for going through the chapters and
+watching the project grow.
+
+On the whole, the chapter-specific exercises follow directly from the
+chapter content and provide good practice opportunities. In certain
+cases, an exercise requires executing a complex set of steps (ex: 3.8.3,
+4.11.2, or 9.8.6) and could be more valuable if there were hints and/or
+scaffolding to help less experienced readers get started.
+
+## Figures and tables
+
+Are the figures and tables visually appealing and useful for the
+reader?
+
+The screenshots in chapter 6 (Advanced Git) are great, and extremely
+helpful in illustrating what the reader might see in using Github.
+
+All of the figures come at appropriate points in the chapter where a
+figure can aid understanding. I found Figures 2.5 and 2.6 hard to
+understand.
+
+There is a note in chapter 5 (Git at the Command Line) about including
+Figure 5.7. I would include this figure (or one like it) and would also
+recommend going back through the chapter and replacing existing diagrams
+to match the format of this final figure, so all the figures in that
+chapter build up to a final summary like 5.7.
+
+## Length
+
+Do you think the proposed length seems appropriate for the topic and
+readership?
+
+Yes. I think the length is appropriate for the topic; it matches other
+"programming" books of my experience. I also think it is a good length
+to be adapted as an intermediate-level semester-length course, or as a
+reference/self-study book for a researcher or other data science
+professional role.
+
+## Improvements
+
+Please outline any suggestions you may have to improve the format,
+layout, and design of the book.
+
+For the callout boxes, it would be helpful for the horizontal lines to
+somehow indicate which side is "inside" the box, and/or for the
+preformatted text to be indented in the same way as the rest of the text
+inside the box. On pages where the callout box spans a page break or
+where there is preformatted text, it can be hard to know which bit is a
+callout box and which is the regular chapter text.
```

Comment:
I'm not 100% sure what this comment is referring to but this could be fixed by using a custom callout block (https://bookdown.org/yihui/bookdown/custom-blocks.html), rather than the blockquotes.

Reply:
I assume she is referring to the PDF document as opposed to the online book? If so, maybe the LaTeX wizard that we're going to use can make the callout boxes look better?

Reply:
@DamienIrving that's correct -- it was 100% the PDF formatting.